reference/checkouts.v3.yml

```diff
@@ -2879,6 +3333,183 @@ definitions:
       - amount
       - sender
       - recipient
+    ItemCustom:
```

Comment:
Can you please verify that custom_items objects return all of these fields? Also, we should review formats and remove `double` for the properties that do not support fractional values.

Reply:
`custom_items` looks correct in Stoplight.